bundles/model/org.eclipse.smarthome.model.core/src/main/java/org/eclipse/smarthome/model/core/internal/folder/FolderObserver.java

```diff
@@ -289,10 +287,8 @@ private static File getFileByFileExtMap(Map<String, String[]> folderFileExtMap,
      *            the file name to get the {@link File} for
      * @return the corresponding {@link File}
      */
-    private static File getFile(String filename) {
-        File folder = new File(ConfigConstants.getConfigFolder() + File.separator + filename);
-
-        return folder;
+    private File getFile(String filename) {
+        return new File(ConfigConstants.getConfigFolder() + File.separator + filename);
```

Comment:
...or 

```java
File(ConfigConstants.getConfigFolder(), filename)
```

Frankly said, I did not even closely look at it, just hit "Cmd+1" -> "inline" 😞